src/base/dom/pan-and-zoom.ts

```diff
@@ -35,6 +37,19 @@ export class PanAndZoom {
         public max_zoom = 10,
         public bounds?: BBox,
     ) {
+        Preferences.INSTANCE.addEventListener(
+            "kicanvas:preferences:change",
+            () => {
+                //this.updateSettings(Preferences.INSTANCE);
+                console.log(
+                    "Preferences changed:",
+                    prefs.alignControlsWithKiCad,
+                );
+            },
+        );
+
+        //this.updateSettings(Preferences.INSTANCE);
```

Comment:
If this code is supposed to be here, it should be uncommented. If not, it should just be removed.

Reply:
I removed this part as well because it was not necessary.